iOS/PartySample/app/inc/pch.h

```diff
@@ -1,3 +1,4 @@
+#include <assert.h>
```

Comment:
You don't introduce assert in this PR. Why we need to add this?

Reply:
The code was not building without it, unsure what changed, I see also that the Windows pch has it.

Reply:
Then `#include <cassert>` is preferred style in your first commit.

Reply:
Agreed, but the Windows project uses assert.h https://github.com/PlayFab/PlayFabParty/blob/1eb0ad7a0bc77b873626b12d72a359892fb8abc8/win32/PartySample/dll/pch.h#L10 so I kept it for consistency.